packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -0,0 +1,393 @@
+```
+
+#### Usage
+
+```jsx
+import {MultiselectCombobox} from '@twilio-paste/core/combobox';
+
+const items = [
+  'Alert',
+  'Anchor',
+  'Button',
+  'Card',
+  'Heading',
+  'List',
+  'Modal',
+  'Paragraph',
+  'Popover',
+  'Tooltip',
+];
+
+function getFilteredItems(inputValue) {
+  const lowerCasedInputValue = inputValue.toLowerCase();
+
+  return items.filter(function filterItems(item) {
+    return item.toLowerCase().includes(lowerCasedInputValue);
+  });
+}
+
+const MyComponent = () => (
+  const [inputValue, setInputValue] = React.useState('');
+  const filteredItems = React.useMemo(() => getFilteredItems(inputValue), [inputValue]);
+
+  return (
+    <MultiselectCombobox
+      labelText="Choose a Paste Component"
+      selectedItemsLabelText="Selected Paste components"
+      items={filteredItems}
+      onInputValueChange={({inputValue: newInputValue = ''}) => {
+        setInputValue(newInputValue);
+      }}
+    />
+  );
+);
+```
+
+#### Component props
+
+| Prop                     | Type                                      | Description                                                                                        | Default                                                                   |
+| ------------------------ | ----------------------------------------- | -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| `labelText`              | `string \| NonNullable<React.ReactNode>`  | The contents of the label text.                                                                    |                                                                           |
+| `selectedItemsLabelText` | `string`                                  | Hidden helper text for screen readers                                                              | `''`                                                                      |
+| `groupItemsBy?`          | `string`                                  | The name of the key in your item objects that you would like Combobox to group the items by.       |                                                                           |
+| `groupLabelTemplate?`    | `(groupName: string) => React.ReactNode`  | This function allows you to use your own `jsx` template for the group label in the drop-down list. |                                                                           |
+| `helpText?`              | `string \| React.ReactNode`               | The contents of the help and error text.                                                           |                                                                           |
+| `i18nKeyboardControls?`  | `string`                                  | Visually hidden string that has instructions for how to remove and select pills with a keyboard.   |                                                                           |
```

Comment:
issue: the `i18nKeyboardControls` prop is listed twice. I'd delete this one and keep the one below, since it has the correct default value for the string